app/renderer/ui/components/main/sections/wallet/tabs/transactions/index.tsx

```diff
@@ -36,29 +41,43 @@ class Transactions extends TabComponent<any> {
       }
     ))
 
+    const pendingTransactionsList = (
+      <List
+        dataSource={this.props.pendingTransactions}
+        renderItem={PendingTransaction}
+        emptyIcon="generic"
+        emptyText="You don't have pending transactions"
+      />
+    )
+
+    const confirmedTransactionsList = (
+      <List
+        dataSource={this.props.confirmedTransactions}
+        renderItem={ConfirmedTransaction}
+        emptyIcon="generic"
+        emptyText="You don't have confirmed transactions"
+      />
+    )
+
     return (
       <>
         <div className={styles.left}>
           <Wrapper
             title="PENDING"
-            caption="2 transactions"
+            caption={`${Object.keys(this.props.pendingTransactions).length} transactions`}
             className={styles.pending}
+            empty={!Object.keys(this.props.pendingTransactions).length}
           >
-            <List
-              dataSource={pendingTransactions}
-              renderItem={PendingTransaction}
-            />
+            {pendingTransactionsList}
           </Wrapper>
           <Wrapper
             title="CONFIRMED"
-            caption="3 transactions"
+            caption={`${Object.keys(this.props.confirmedTransactions).length} transactions`}
```

Comment:
Why couldn't this just be the length of the `this.props.confirmedTransactions` array?